content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -72,7 +72,7 @@ and that the certificate contains the configured DNS names, you need to get the
 decode it, and view its text output with the following command:
 
 {{< text bash >}}
-$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in /dev/stdin -text -noout
```

Comment:
Why is this change needed? Why can't `-` be used here?

Reply:
I think I made a mistake, this pull request is based on the master branch, soon after I submitted another patch with the dns-cert/index.md file changed to the master branch, it is also appeared here, I have submitted a new patch based on the newly created branch https://github.com/istio/istio.io/pull/6282, I will close this one soon.